<launch>
  <arg name="debug" default="false"/>
  
  <!-- Launch the tactile sensor -->
  <!--node pkg ="rosserial_python" type="serial_node.py" name="serial_node" args="/dev/ttyACM0"/>
  <include file="$(find tactile_pressure_sensor)/launch/run_tactile_pressure.launch" /-->
  
  <!-- Launch the Robotiq 2f gripper -->
  <node pkg="robotiq_2f_gripper_control" type="Robotiq2FGripperRtuNode.py" name="Robotiq2FGripperRtuNode" args = "/dev/ttyUSB0" />
  
  <!-- Launch the gripper control loop >
  <node pkg="fm_gripper_module" type="Robotiq2FGripperSlipController.py" name="Robotiq2FGripperSlipController" /-->
  
  <!-- Launch the action server -->
  <node pkg="fm_gripper_module" type="gripper_action_server.py" name="gripper_action_server" />
  
</launch>
